bss.cc

```diff
@@ -555,6 +555,35 @@ trefethen4(const std::vector<double>& xs)
     return ret;
 }
 
+//Valley Functions
+double
+threeHumpCamel(const std::vector<double>& xs)
+{
+    double x1 = xs.at(0);
+    double x2 = xs.at(1);
+
+    double ret = (2 * SQ(x1)) 
```

Comment:
for the style consistency, I'd recommend using " \" at the end of a line

Reply:
Sorry, I don't understand this. What do you mean by adding " "?

Reply:
this is my bad. I thought about adding " \" as in <space><backslash>.

Reply:
Ah gotchu you're good!

Reply:
umm I think I keep running into some special symbols.. 
But I was mistaken thinking that continuations are necessary in C++. It applies to preprocessor directives, but not to statements, so we are good with this one.

I meant there..
`<SPACE><BACKSLASH>`.

(hopefully that won't get interpreted somehow..)